controllers/v1/library/categoriesController.js

```
@@ -86,4 +87,109 @@ module.exports = class LibraryCategories extends Abstract {
+					req.files,
+					req.userDetails
+				)
+
+				return resolve({
+					message: libraryProjectcategory.message,
+					result: libraryProjectcategory.data,
+				})
+			} catch (error) {
+				return reject(error)
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)